deployment/build_and_release/modules/release/main.tf

```diff
@@ -235,4 +235,232 @@ resource "google_cloudbuild_trigger" "release" {
   filename = "${each.value.cloudbuild_path}"
 }
 
+resource "google_cloudbuild_trigger" "os_build" {
```

Comment:
Is it worth sticking a TODO on line 218 to make that similar to this one (and rename is to applet_build) ?

And thinking about this led me to wonder why the other is called `release` and this is `build`? Are they both really `build_and_release` ?

Reply:
Cool, that sounds good to me.